mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -141,6 +145,18 @@ Iterable<MapEntry<String, Map<String, dynamic>>> _listJobs(
       );
     }
   }
+
+  // Generate the jobs that run on completion of all other jobs, by adding the
+  // appropriate `needs` config to each.
+  if (onCompletionJobs != null && onCompletionJobs.isNotEmpty) {
+    final needs = List.generate(count, (i) => _jobName(i + 1));
+    for (var jobConfig in onCompletionJobs) {
+      yield _jobEntry({
```

Comment:
at least an assert on not having `needs` already?

Reply:
I am adding that now, its still a draft!

Reply:
Oh, I forgot to mark it as draft.... lol